ratbagd/ratbagd-device.c

```diff
@@ -137,6 +137,29 @@ static int ratbagd_device_get_device_name(sd_bus *bus,
 	return 0;
 }
 
+static int ratbagd_device_get_device_type(sd_bus *bus,
+					  const char *path,
+					  const char *interface,
+					  const char *property,
+					  sd_bus_message *reply,
+					  void *userdata,
+					  sd_bus_error *error)
+{
+	struct ratbagd_device *device = userdata;
+	enum ratbag_device_type devicetype;
+
+	devicetype = ratbag_device_get_device_type(device->lib_device);
+	if (!devicetype) {
+		log_error("%s: device type unspecified\n",
+			  ratbagd_device_get_sysname(device));
+		devicetype = TYPE_UNSPECIFIED;
```

Comment:
This is a no-op because `!devicetype` is equal to `devicetype == 0` and `TYPE_UNSPECIFIED` is `0`.